docker/store.sh

```diff
@@ -49,11 +58,33 @@ function image_variant() {
 
 store_image() {
   local image="$1"
-  docker tag "$(image_variant "${image}")" "$(image_variant "${image}" "${sha}")"
-  docker tag "$(image_variant "${image}")" "$(image_variant "${image}" "${branch_tag}")"
+  local platforms="${2}"
+
+  sha_image=$(image_variant "${image}" "${sha}")
+  branch_image=$(image_variant "${image}" "${branch_tag}")
+  latest_image=$(image_variant "${image}" latest)
+
+  if [ -n "${platforms}" ]
+  then
+    manifest_list=()
+    while IFS=, read -r platform
+    do
+      platform_image=$(image_variant "${image}" "${platform}")
+      import_image "${platform_image}" "${DOCKER_IMAGE_CACHE}"
```

Comment:
Does `.platforms` always exist, even for a single platform?
If no, then doesn't `import_image` get skipped now in the single platform case?
If yes, then can the code in `build.sh` not be written in a similar fashion, where there would always be a list of platforms and so we don't do the [if/else](https://github.com/product-os/scripts/pull/157/files#diff-efe74abc7213e95246105b4a7b029557ef16ff997e9cc13eadcfffa18e6757c0R227)?

Reply:
Good catch, I realized that this morning too. Platform will not always exist, I want to make sure the current behaviour doesn't change if platform isn't provided, at least for now. Maybe we will migrate all builds to buildx at a later date.

Will fix this up.